### PR TITLE
Added comments to gethelpnow-style.css

### PR DESCRIPTION
**Pull request by kamsuri**
Added comments to gethelpnow-style.css
Fixed issue #22

### CODE REVIEW

**Comment by shreyans29** [comment]
@AkankshaBodhankar  Since this has been pending from 25 days now, is it okay if I can complete it ?

**Comment by AkankshaBodhankar** [comment]
No. we will wait until she replies.

File: css files/gethelpnow-style.css [hunk 1]
```diff
@@ -40,14 +35,18 @@ body {
 	width: 400px;	
 }
 
+/* ===BUTTONS=== */
+
+/* Buttonf for each sub-feature */
```
Comment (kamsuri) [inline css files/gethelpnow-style.css]: Sorry that was a typing error.

**Comment by kamsuri** [comment]
Sorry for the delayed response. I have submitted a new PR with all the above mentioned issues resolved. I was not able to make new commits to this PR due to some error.

**Comment by kamsuri** [comment]
@AkankshaBodhankar can you also tell me if i can contribute to this project for GSOC'17?

**Comment by fatimarafiqui** [comment]
Updated PR #202 .
@AkankshaBodhankar @MehaKaushik Let's close this PR.

**Pull request closed without merging**